docs/plugins/virtual_environment_docs/2-deploying-your-plugin.md

```diff
@@ -0,0 +1,109 @@
+# Deploying your plugin  
+
+This guide explains some of the techniques you can use to deploy your plugin. 
+
+## This guide covers:   
```

Comment:
Hmm this structure is a little bit confusing. We only really talk about Anaconda in the very last bit, so I would make that its own heading, and focus on PyPI in the first two sections. So this would be my suggested structure:

```
* PyPI package formats
    - Overview
    - Building your package
* Deploying to PyPI
    - Manually via twine
    - Automatically via GitHub actions
* Deploying to Anaconda
```

Reply:
Good idea.

Reply:
Ok, @DragaDoncila, please see what you think now.